website/client/src/components/chat/chatMessages.vue

```diff
@@ -303,7 +294,7 @@ export default {
         container.style.overflowY = 'scroll';
       }
     }, 50),
-    messageLiked (message) {
+    async messageLiked (message) {
```

Comment:
Why async? There's nothing `await`ed in here :thinking:

Reply:
probably a mix up when I merged the chatMessage component of normal and private chats together - yeah not needed at all- removing